lib/screens/wh/goods_dispatch.dart

```diff
@@ -647,6 +697,10 @@ String qtyToTextInner(String text, Map qty) {
       }
     }
   }
+  // workaround
+  if (text.characters.first == ' ') {
```

Comment:
give better description for season of workaround

Reply:
trim always delete spaces, no need for check